api/game.py

```diff
@@ -316,3 +408,27 @@ def get_player_info(player_names):
 
     data["players"] = [p.name for p in players]
     return data
+
+
+def main():
+    players = ["Alice", "Bob", "Charlie", "David", "Eve", "Frank"]
+    num_players = len(players)
+    if not (5 <= num_players <= 10):
+        raise "Invalid number of players"
+
+    players = set(players)  # use as set to avoid duplicate players
+    players = list(players)  # convert to list
+    random.shuffle(
+        players
+    )  # ensure random order, though set should already do that
+    if len(players) != num_players:
+        raise "Duplicate player names"
+
+    computedData = get_player_info(players)
+    print(computedData["doNotOpen"])
+    for player in players:
+        print(computedData[player])
+
+# uncomment to run the main function for testing
+# please leave commented out when deploying
+# main()
```

Comment:
Does `if __name__ == "__main__":` not work here?

Reply:
not sure :/